FoodplannerApi/Program.cs

```diff
@@ -202,6 +206,7 @@
 builder.Services.AddScoped<IFoodImageService, FoodImageService>();
 builder.Services.AddScoped<IChatService, ChatService>();
 builder.Services.AddScoped<IPasswordHandler, PasswordHandler>();
+builder.Services.AddScoped(typeof(IOneTimePasswordService), typeof(OneTimePasswordService));
```

Comment:
Don't know why we register services and repositories differently, but we should try to keep it consistent even if it is goofy:
```suggestion
builder.Services.AddScoped<IOneTimePasswordService, OneTimePasswordService>();
```